src/logic/schedule-parser/schedule.parser.ts

```diff
@@ -4,9 +4,24 @@ import { ChildrenInfoParser } from "./children-info.parser";
 import { DataRowParser } from "./data-row.parser";
 import { MetaDataParser } from "./metadata.parser";
 import { ShiftsInfoParser } from "./shifts-info.parser";
-import { ScheduleProvider, Schedule, Sections } from "../providers/schedule-provider.model";
+import { Schedule, ScheduleProvider, Sections } from "../providers/schedule-provider.model";
 import { ExtraWorkersParser } from "./extra-workers.parser";
-import { MetaDataRowLabel } from "../section.model";
+import { ChildrenSectionKey, MetaDataRowLabel, MetaDataSectionKey } from "../section.model";
+
+export enum MONTHS {
```

Comment:
Mamy już coś podobnego. 
`nurse-scheduling-problem-frontend/src/helpers/tranlsations.helper.ts`
```
      styczeń: "january",
      luty: "february",
      marzec: "march",
      kwiecień: "april",
      maj: "may",
      czerwiec: "june",
      lipiec: "july",
      sierpień: "august",
      wrzesień: "september",
      październik: "october",
      listopad: "november",
      grudzień: "december",
```
Więc proponuję lub zamienić we wszystkich miejscach w których używamy tego obiektu, użyć enum który napisałaś ty, lub w tym miejscu użyć tego obiektu.
 Jbc, to mi odpowiadają obie opcje ;)